docs/02-app/02-api-reference/05-next-config-js/images.mdx

```diff
@@ -157,11 +157,10 @@ export default function thumborLoader({ src, width, quality }) {
 
 ```js
 // Docs: https://supabase.com/docs/guides/storage/image-transformations#nextjs-loader
-const projectId = '' // your supabase project id
-
 export default function supabaseLoader({ src, width, quality }) {
-  return `https://${projectId}.supabase.co/storage/v1/render/image/public/${src}?width=${width}&quality=${
```

Comment:
For supabase, the storage url has to be of this specific shape. Could you revert the change to make that clear? See the docs here: Docs: https://supabase.com/docs/guides/storage/image-transformations#nextjs-loader

Reply:
@styfle fyi ☝️

Reply:
All of the examples expect the prefix to be replaced. I think if we do this for supabase then we should change it for all of them. What I'm not sure is if there is a consistent prefix for each provider. Also what is the difference between `v1` and `v2`, for example?

Reply:
> I think if we do this for supabase then we should change it for all of them.

Hm, yah, maybe a good idea.

> Also what is the difference between v1 and v2, for example?

Currently we only have v1, it's just a placeholder in case there will be breaking changes in the future.